sdk/textanalytics/Azure.AI.TextAnalytics/src/JobManifestTasks.cs

```diff
@@ -16,17 +16,17 @@ public partial class JobManifestTasks
         /// <summary>
         /// EntityRecognitionTasks
         /// </summary>
-        public IList<EntitiesTask> EntityRecognitionTasks { get; }
+        public IList<EntitiesTask> EntityRecognitionTasks { get; internal set; }
```

Comment:
whenever possible, we avoid making collections settable. You can create a `JobManifestTask` constructor that takes as parameters the tasks. i.e. `JobManifestTasks(entitiesTask, piiTask, KeyPhrasesTask)`

Reply:
I am making them internally settable as there will be cases when we do not want all 3 to be set through one constructor. there can be cases when only 1 or 2 need to be set.

Reply:
The thing I am worried about too is that the user is now able to create a `JobManifestTask` object and not set any task and send that. By having the option in the constructor, the user can't create a bad instance.

One way to do it might be by using default values in the constructor.
i.e. 
```
JobManifestTask(IReadOnlyList<EntitiesTask> entityRecognitionTasks = default, .... )
```